fenix/app/src/main/java/org/mozilla/fenix/translations/preferences/downloadlanguages/PrefDownloadLanguageItem.kt

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations.preferences.downloadlanguages
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.mozilla.geckoview.TranslationsController
+
+/**
+ * PrefDownloadLanguageItem that will appear on [PrefDownloadLanguagesFragment] screen.
+ *
+ * @property languageModel The object comes from the translation engine.
+ * @property state State of the [languageModel].
+ */
+data class PrefDownloadLanguageItem(
+    val languageModel: TranslationsController.RuntimeTranslation.LanguageModel,
+    val state: PrefDownloadLanguageItemState,
+)
+
+/**
+ * PrefDownloadLanguageItemState the state of the [PrefDownloadLanguageItem]
+ *
+ * @property type The type of the language item.
+ * @property status State of the language file.
+ */
+@Parcelize
+data class PrefDownloadLanguageItemState(
+    val type: PrefDownloadLanguageItemType,
+    val status: PrefDownloadLanguageItemStatus,
+) : Parcelable
+
+/**
+ * PrefDownloadLanguageItemType the type of the [PrefDownloadLanguageItem]
+ */
+enum class PrefDownloadLanguageItemType {
+    EnglishLanguage,
```

Comment:
nit: Please change to `PivotLanguage` as we could potentially have other pivot languages besides English in the future.

Reply:
Done